packages/ui/src/components/ExitIntentModal.tsx

```
@@ -1,145 +1,188 @@
-      className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50 transition-opacity duration-300"
-      onClick={handleBackdropClick}
-      role="dialog"
-      aria-modal="true"
-      aria-labelledby="exit-intent-title"
-      aria-describedby="exit-intent-description"
-    >
+    <div className="fixed inset-0 z-50 flex items-center justify-center p-4">
+      <div
+        className="fixed inset-0 bg-black bg-opacity-50 transition-opacity duration-300"
+        onClick={handleBackdropClick}
+      />
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)